server/commands/hubs.py

```diff
@@ -122,14 +125,14 @@ def ooc_cmd_save_hub(client, arg):
                     "Server storage full! Please contact the server host to resolve this issue."
                 )
             try:
+                if os.path.isfile(f"storage/hubs/read_only/{derelative(args[0])}.yaml"):
+                    raise ArgumentError(f"Hub {args[0]} already exists and it is read-only!")
+                if os.path.isfile(f"storage/hubs/{derelative(args[0])}.yaml") and len(args) > 2 and args[2].lower() == "read_only":
```

Comment:
`/save_hub args[0] args[1] args[2]`, args[2] doesn't exist.
this should be `len(args) > 1 and args[1].lower()`, please check all instances of this